tests/update_reference_data.sh

```diff
@@ -130,6 +130,7 @@ tests[udq_pyaction]="flow udq_actionx PYACTION_WCONPROD udq_pyaction"
 tests[spe1_foam]="flow spe1_foam SPE1FOAM"
 tests[wsegsicd]="flow wsegsicd TEST_WSEGSICD"
 tests[bc_lab]="flow bc_lab BC_LAB"
+tests[pinch_multz_all]="flow pinch PINCH_MULTZ_ALL pinch_multz_all"
```

Comment:
this is wrong. why do you give pinch_multz_all at the end?

Reply:
Well; the real answer is of course because I don't know better. I am anticipating another test located in the same `pinch/` directory and then thought this was the way to do it.

Reply:
it is, but then you have to have the test name matching, refer to the comment above (which you wrote! :))

Reply:
:-)